packages/ringcentral-widgets/components/CallsListPanel/index.js

```diff
@@ -175,6 +181,22 @@ export default class CallsListPanel extends Component {
     this.forceUpdate();
   }
 
+  // componentDidUpdate(prevProps, prevState) {
```

Comment:
Please delete unwanted code sections like this, we can always use git commit history to get these code back.